test/app/config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  module: {
+    loaders: [
+      {test: /\.json$/, loader: 'json'}
```

Comment:
Yes, the `entities` one, required by `commonmark` or `commonmark-react-renderer`.